src/data/tms.ts

```diff
@@ -31,7 +32,7 @@ export const reverseCompatibleTms: MoveId[] = []; /*[
     MoveId.ROUND
 ];*/
 
-export const tmSpecies: TmSpecies = {
+export const tmSpecies: Readonly<TmSpecies> = deepFreeze({
```

Comment:
NIT: Can be simplified by using the generic type _argument_

```suggestion
export const tmSpecies = deepFreeze<TmSpecies>({
```